Sources/TokamakCore/Styles/TextFieldStyle.swift

```diff
@@ -15,30 +15,60 @@
 //  Created by Jed Fox on 06/30/2020.
 //
 
-public protocol TextFieldStyle {}
+public struct _TextFieldStyleLabel: View {
+  public let body: AnyView
+}
+
+public protocol TextFieldStyle: _AnyTextFieldStyle {
```

Comment:
@carson-katri is this `_AnyTextFieldStyle` refactoring somehow required for the toolbar stuff to work? If not, what's its purpose? Could we maybe move it to a separate PR?

Reply:
A `ToolbarTextFieldStyle` is defined/used here: `Sources/TokamakStaticHTML/Views/Navigation/NavigationView.swift`